### PR TITLE
chore: create user_story issue template

### PR DESCRIPTION
**Pull request by pbiology**
### This PR:

Added: A User Story issue template using the GitHub issue forms

### Review and tests: 
N/A

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/user_story.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+name: User Story
+description: A user story for BALSAMIC
+title: "[User Story] <title>"
+labels: ["User-Story"]
```
Comment (ivadym) [inline .github/ISSUE_TEMPLATE/user_story.yml:4]: Are we switching from `feature` labels to `User-Story`?

**Review by ivadym — APPROVED** [review]
Looks good 👍

File: .github/ISSUE_TEMPLATE/user_story.yml (hunk 1 above)
Comment (pbiology) [reply .github/ISSUE_TEMPLATE/user_story.yml:4]: yeah. I know that's a little bit annoying, but it will harmonise with other projects

Comment (pbiology) [reply .github/ISSUE_TEMPLATE/user_story.yml:4]: And the other aspect is that "feature" is a word used in other contexts elsewhere

**Comment by pbiology** [comment]
Bypassing the changelog check as this does not affect the code-base at all

File: .github/ISSUE_TEMPLATE/user_story.yml (hunk 1 above)
Comment (ivadym) [reply .github/ISSUE_TEMPLATE/user_story.yml:4]: Sounds good 👍

**Pull request merged**